### PR TITLE
javaTpoint js tutorial is added

### PR DESCRIPTION
**Pull request by Sushilverma002**
close issue #1329 

## Screenshots
image

### CODE REVIEW

**Comment by Sushilverma002** [comment]
@CBID2 have a look!!!!!!!!!!!!

File: database/languages/javascript.json [hunk 1]
```diff
@@ -68,5 +68,12 @@
     "url": "https://javascript30.com/",
     "category": "languages",
     "subcategory": "javascript"
+  },
+  {
+    "name": "javaTpoint",
+    "description": "Our JavaScript Tutorial caters to beginners and professionals, utilizing lightweight, cross-platform object-based scripting for creating dynamic client-side pages using the JavaScript Translator.",
```
Comment (CBID2) [inline database/languages/javascript.json:74]:
```suggestion
    "description": "Our JavaScript tutorial caters to beginners and professionals, utilizing lightweight, cross-platform object-based scripting for creating dynamic client-side pages using the JavaScript Translator.",
```

**Review by CBID2 — CHANGES_REQUESTED** [review]
Found a typo @Sushilverma002

**Comment by Sushilverma002** [comment]
@CBID2 now check issue is resolved

**Review by CBID2 — APPROVED** [review]
LGTM! :)

**Comment by Sushilverma002** [comment]
@CBID2 please merge this PR and give level label.

**Comment by CBID2** [comment]
> @CBID2 please merge this PR and give level label.
> 
> 

Two approvals are required for merging. Please be patient

**Comment by Sushilverma002** [comment]
okay for sure.

**Review by k-deepak04 — APPROVED** [review]

**Pull request merged**